app/sidero-controller-manager/config/samples/metal_v1alpha2_environment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: metal.sidero.dev/v1alpha2
+kind: Environment
+metadata:
+  name: default
+spec:
+  kernel:
+    url: "https://github.com/talos-systems/talos/releases/download/v0.10.3/vmlinuz-amd64"
```

Comment:
probably just use latest

Reply:
yep, good point, and siderolabs, I wonder if someone is actually using those samples :shrug: